open_event/static/admin/js/prism.js

```diff
@@ -1,9 +1,717 @@
+
+/* **********************************************
+     Begin prism-core.js
+********************************************** */
+
+var _self = (typeof window !== 'undefined')
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier '_self' does not match the pattern '^[a-z]+([A-Z][a-z]+)*$'.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2574540462/issues/source?bid=3233864&fileBranchId=3298965#l6)